contracts/interfaces/0.8.x/IMintingAPICoreV0.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+interface IEngineContractMintingAPIV0 {
```

Comment:
Can we name this `IMintingAPICoreV0` please?

Reply:
Done 👍